aws/resource_aws_macie2_organization_admin_account_test.go

```diff
@@ -0,0 +1,136 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/macie2"
+	"github.com/hashicorp/aws-sdk-go-base/tfawserr"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func testAccAwsMacie2OrganizationAdminAccount_basic(t *testing.T) {
+	resourceName := "aws_macie2_organization_admin_account.test"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
```

Comment:
Because we're creating an Organization, we have to add a `PreCheck` that ensures we're not already in an Organization account

```suggestion
		PreCheck: func() {
			testAccPreCheck(t)
			testAccOrganizationsAccountPreCheck(t)
		},
```